airflow/providers/hashicorp/provider.yaml

```diff
@@ -55,6 +55,7 @@ versions:
 dependencies:
   - apache-airflow>=2.6.0
   - hvac>=1.1.0
+  - apache-airflow-providers-amazon>=8.0.0
```

Comment:
Should (or could) we make this an extra dep for those who need to use aws?

Reply:
removed `apache-airflow-providers-amazon` and added `boto3` in extra PTAL